frontend/src/constants/datasets/TX_texas_abortion_clinics_address_mar2017.json

```diff
@@ -1,9 +1,9 @@
 {
     "dataSources": "<a href=\"https://fundtexaschoice.org/index.php/about/\" target=\"_blank\">Fund Texas Choice</a>, Feb 2018.",
-    "description": "The passing of Texas House Bill 2 closed nearly 75% of abortion clinics in rural and low-income areas between 2013 and 2014. Although HB2 was overturned in 2016, many clinics have not re-opened. Use this map to explore the accessibility of existing clinics.",
+    "description": "The passing of Texas House Bill 2 closed nearly 75% of abortion clinics in rural and low-income areas between 2013 and 2014. Although HB2 was overturned in 2016, many clinics have not re-opened. Use this map to explore the accessibility of existing clinics in and around Texas.",
     "subtitle": "Explore how maps can illuminate policy impacts",
     "state": "tx",
-    "name": "Abortion Clinics in Texas",
+    "name": "Abortion Clinics Near Texas",
```

Comment:
Near sounds excluding of Texas? But idk....

Reply:
It's from the ticket description.